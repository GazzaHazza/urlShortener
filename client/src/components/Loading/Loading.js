import React, { Component } from "react";
import "./Loading.css";
class Loading extends Component {
  render() {
    return (
      <div className="loading">
        <span className="fa fa-spinner fa-pulse loading__icon" />
      </div>
    );
  }
}
export default Loading;
