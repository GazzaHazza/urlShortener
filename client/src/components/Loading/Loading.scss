.loading {
    position: absolute;
    width: 250px;
    height: 250px;
    background: rgba(234,234,234, .8);
    left: calc(50% - 125px);
    top: calc(50% - 75px);
    .loading__icon {
        font-size: 100px;
        text-align: center;
        display: block;
        margin-top: 74px;
    }
}