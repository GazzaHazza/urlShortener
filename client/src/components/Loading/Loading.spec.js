import React from "react";
import Loading from "./Loading";
import { shallow } from "enzyme";
import "jest-styled-components";

it("renders correctly", () => {
  const wrapper = shallow(<Loading />);
  expect(wrapper).toMatchSnapshot();
});
