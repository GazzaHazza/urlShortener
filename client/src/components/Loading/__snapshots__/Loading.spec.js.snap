// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="loading"
>
  <span
    className="fa fa-spinner fa-pulse loading__icon"
  />
</div>
`;
